AutoFinance.Broker/InteractiveBrokers/Controllers/ITwsControllerBase.cs

```diff
@@ -173,5 +179,45 @@ public interface ITwsControllerBase
             string exchange,
             string underlyingSecType,
             int underlyingConId);
+
+        /// <summary>
+        /// Set the type for the market data feed
+        /// </summary>
+        /// <param name="marketDataTypeId">The feed level</param>
+        void RequestMarketDataType(int marketDataTypeId);
```

Comment:
Add an integration test for at least `RequestMarketDataType` to try to keep our implemented API surface covered.